package.json

```diff
@@ -994,6 +1013,12 @@
         }
       ]
     },
+    "submenus": [
+      {
+        "id": "mdb.copilot",
+        "label": "MongoDB Copilot"
```

Comment:
Good concern. I was following the design from @bsradcliffe. @GaurabAryal, Ben what do you think? We do need to indicate somehow that the result of the command will be generated by AI. Maybe we can name it `MongoDB Participant`?

Reply:
Great catch, Nikola! Alena, my apologies; the design was not updated per the discussion we had in Slack via a thread. Gaurab clarified [here](https://mongodb.slack.com/archives/C07K1K898J1/p1725029444221269?thread_ts=1724954681.858029&cid=C07K1K898J1), and it's been reflected back into the design/prototype [here](https://www.figma.com/proto/n73EW9wKUG0s5kBNDQoBOA/VS-Code-Export-to-Language?page-id=0%3A1&node-id=8-625&node-type=canvas&viewport=287%2C25%2C0.17&t=YMh2gIp7LxrDzF7G-1&scaling=contain&content-scaling=fixed&starting-point-node-id=8%3A625&show-proto-sidebar=1).

Reply:
Thx Ben! Another question, how do you think about renaming "Run Query in Playgrounds" to "Export Code to Playground"? I feel like "export" passes better rather than "run" because we do not run immediately, we just open a playground with an inserted text.